.github/workflows/compilation_on_sgx.yml

```diff
@@ -140,6 +140,7 @@ jobs:
             # "-DWAMR_BUILD_SIMD=1",
             "-DWAMR_BUILD_TAIL_CALL=1",
             "-DWAMR_DISABLE_HW_BOUND_CHECK=1",
+            "-DWAMR_BUILD_SGX_IPFS=1",
```

Comment:
How about also add compiling and running the sample in the related CI files:

https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/.github/workflows/compilation_on_android_ubuntu.yml#L409-L415

https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/.github/workflows/compilation_on_macos.yml#L352-L358

https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/.github/workflows/compilation_on_sgx.yml#L359-L365

Reply:
I've added a lightweight `iwasm` runner to the sample to execute it in the CI you linked.